bin/make_reference_catalog.py

```diff
@@ -8,10 +13,10 @@
 from lsst.sims.catUtils.mixins import AstrometryStars, PhotometryStars
 
 class TwinklesReference(InstanceCatalog, AstrometryStars, PhotometryStars):
+    "Reference stars for Twinkles astrometry."
     catalog_type = 'twinkles_ref_star'
-    column_outputs = ['uniqueId', 'raJ2000', 'decJ2000', 'lsst_g',
-                      'lsst_r', 'lsst_i', 'starnotgal',
-                      'isvariable']
+    column_outputs = ['uniqueId', 'raJ2000', 'decJ2000', 'lsst_u', 'lsst_g',
+                      'lsst_r', 'lsst_i', 'lsst_z', 'lsst_y', 'isvariable']
     default_columns = [('isresolved', 0, int), ('isvariable', 0, int)]
```

Comment:
For the new system we want isresolved, so I think it should be that starnotgal should be changed to isresolved in line 13 rather than removing that column completely.